lib/ui/form/widget/form_survey_answer_smiley.dart

```diff
@@ -16,25 +16,29 @@ class FormSurveyAnswerSmiley extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final emojiState = ref.read(selectedEmojiProvider.notifier);
-    return Row(
-      mainAxisAlignment: MainAxisAlignment.center,
-      children: [
-        for (final emoji in emojis) ...[
-          GestureDetector(
-            onTap: () {
-              emojiState.state = emoji;
-            },
-            child: Padding(
-              padding: const EdgeInsets.all(AppDimensions.spacing8),
-              child: buildEmoji(emoji, ref),
+    return Container(
+      width: double.infinity,
+      height: AppDimensions.answerSmileyHeight,
+      alignment: Alignment.center,
+      child: ListView(
+        scrollDirection: Axis.horizontal,
+        shrinkWrap: true,
+        children: [
+          for (final emoji in _emojis) ...[
+            GestureDetector(
+              onTap: () => emojiState.state = emoji,
+              child: Padding(
+                padding: const EdgeInsets.all(AppDimensions.spacing8),
+                child: _buildEmoji(emoji, ref),
+              ),
             ),
-          ),
-        ]
-      ],
+          ]
+        ],
+      ),
```

Comment:
Should we use [ListView.builder](https://api.flutter.dev/flutter/widgets/ListView-class.html)?